File: 242-ValidAnagram/242-ValidAnagram.java
// Last updated: 7/28/2025, 6:14:30 PM
class Solution {
    public boolean isAnagram(String s, String t) {
        char[] sChars = s.toCharArray();
        char[] tChars = t.toCharArray();

        Arrays.sort(sChars);
        Arrays.sort(tChars);
        
        return Arrays.equals(sChars, tChars);
        
    }
}

File: 242-ValidAnagram/242-ValidAnagram.py
# Last updated: 7/28/2025, 6:14:37 PM
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        s1 = sorted(s)
        s2 = sorted(t)
        if s1 == s2:
            return True
        else:
            return False